CONTRIBUTING.md

```diff
@@ -19,7 +27,7 @@ type(scope): subject
 Besides the version bump, the changelog and release notes are formatted accordingly.
 So based on the example above:
 
-> ### Documentation
+> <h3>Documentation</h3>
 > 
```

Comment:
This is the workaround when quoting text with headers.  Really, `markdown-toc` should handle this but it would mean maintaining our own fork.